scripts/install-build-tools.sh

```diff
@@ -6,19 +6,74 @@ green="\033[0;32m"
 cyan="\033[0;36m"
 end="\033[0m"
 
-set -e
+# for debugging, 'set -x' can be added to trace the commands
+set -ex
```

Comment:
`-x` is great for debugging, but probably shouldn't be merged in.

Reply:
ACK - changed

Reply:
this is not resolved.

Reply:
It is now, had it back on for debugging thx